recipes/cabal/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+export CFLAGS="-I$PREFIX/include:$CFLAGS" 
+export LDFLAGS="-L$PREFIX/lib:$LDFLAGS"
+export LD_LIBRARY_PATH="$PREFIX/lib:$LD_LIBRARY_PATH"
+export LIBRARY_PATH="$PREFIX/lib:$LIBRARY_PATH"
+export C_INCLUDE_PATH="$PREFIX/include:$C_INCLUDE_PATH" 
+ghc-pkg recache
+cd cabal-install
+export EXTRA_CONFIGURE_OPTS="--extra-include-dirs=$PREFIX/include --extra-lib-dirs=$PREFIX/lib $EXTRA_CONFIGURE_OPTS";
+echo $EXTRA_CONFIGURE_OPTS
```

Comment:
This can be removed?

Reply:
I will try to cut down on the ENV variables